src/App.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const { once } = require('events');
+const express = require('express');
+const config = require('./config.js').config();
+const debug = require('debug');
+
+const info = debug('info');
+const error = debug('error');
+
+class App {
+    constructor(staticPath = '../resources/html') {
```

Comment:
Well, let's remember the questions we should ask yourself before adding a positional argument:

1. **Q**: Is it necessary to obtain this value from each and every user? 
    **A**: I guess not, since this App is not a static or caching web server like Nginx, and you provide your own HTML pages for serving.

2. **Q**: Is it some sort of unique identifier, that should be unique for each user.
    **A**: I guess not, since in 99% you will serve your own HTML pages.

3. **Q**: Can I provide a default value for this parameter?
    **A**: Obviously yes.

I know that you are a Pythonist :) In Python you can omit any argument from any position if it has a default value, and can directly set a parameter by its name. Unfortunately, it's not possible in JS. So we should choose wisely if want to introduce a regular positional argument to our API. If we can answer `Yes` to all three questions above, instead of adding a regular parameter we should use a single object parameter and store actual parameters as properties of the object parameters. It's quite similar to `**kwargs` pattern in Python.
```js
// ❌
function f (optionalArg = 'someValue') {

}

// ✔️
function f ({ optionalArg = 'someValue' } = {}) {

}
```